src/travel-details-screens/components/FlexibleTransportBookingDetails.tsx

```diff
@@ -0,0 +1,201 @@
+  const {t, language} = useTranslation();
+  const style = useStyle();
+  const {theme} = useTheme();
+
+  const {flex_booking_number_of_days_available} = useRemoteConfig();
+  const {configurableLinks} = useFirestoreConfiguration();
+
+  const publicCode = getPublicCodeFromLeg(leg);
+
+  const now = useNow(2500);
+  const bookingIsAvailable = getLegBookingIsAvailable(
+    leg,
+    now,
+    flex_booking_number_of_days_available,
+  );
+
+  const {bottom: safeAreaBottom} = useSafeAreaInsets();
+
+  return (
+    <BottomSheetContainer maxHeightValue={0.83}>
+      <ScreenHeaderWithoutNavigation
+        title={t(
+          TripDetailsTexts.flexibleTransport.needsBookingWhatIsThisTitle(
+            publicCode,
+          ),
+        )}
+        color="background_1"
+        setFocusOnLoad={false}
+      />
+      <TouchableOpacity style={style.closeIconContainer} onPress={close}>
+        <CloseCircle height={theme.icon.size.large} />
+      </TouchableOpacity>
+
+      <ScrollView
+        style={[
+          style.contentContainer,
+          {
+            marginBottom: Math.max(safeAreaBottom, theme.spacings.medium),
+          },
+        ]}
+      >
+        <View style={style.messageBoxContainer}>
+          <FlexibleTransportMessageBox
+            leg={leg}
+            publicCode={publicCode}
+            now={now}
+            showStatusIcon={true}
+          />
+        </View>
+
+        {!isSmallScreen && (
+          <View style={style.imageContainer}>
+            <FlexibleTransport height={107} />
+          </View>
+        )}
+
+        <ThemeText type="heading__title" style={style.title}>
+          {t(TripDetailsTexts.flexibleTransport.contentTitle(publicCode))}
+        </ThemeText>
+        <View style={style.steps} accessibilityRole="list">
+          {
+            // eslint-disable-next-line rulesdir/translations-warning
+            TripDetailsTexts.flexibleTransport.steps.map((step, i) => {
+              const stepNumberText = `${i + 1}. `;
+              const stepInstructionText = t(step);
+              return (
+                <View
+                  key={i}
+                  style={style.step}
+                  accessible={true}
+                  accessibilityLabel={stepNumberText + stepInstructionText}
+                  accessibilityRole="text"
+                >
+                  <ThemeText type="body__primary">{stepNumberText}</ThemeText>
+                  <ThemeText type="body__primary">
+                    {stepInstructionText}
+                  </ThemeText>
+                </View>
+              );
+            })
+          }
+        </View>
+
+        <TouchableOpacity
+          style={style.readMoreAbout}
+          onPress={() => {
+            const flexTransportInfoUrl =
+              getTextForLanguage(
+                configurableLinks?.flexTransportInfo,
+                language,
+              ) || '';
+            flexTransportInfoUrl && Linking.openURL(flexTransportInfoUrl);
```

Comment:
I think no matter what, here `Linking.openURL(flexTransportInfoUrl)` will be called because of the `|| ''`